pages/portfolio.html

```diff
@@ -4,9 +4,50 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Projects</title>
 </head>
-<body>
-  
+<body style="background-color: lightsalmon;">
+  <header>
+    <nav>
+    <a href="./index.html">Home</a>
+    <a href="./about.html">About</a>
+    <a href="./portfolio.html">Projects</a>
+    <link href="style.css" rel="stylesheet">
+    </nav>
+  </header>
+  <h1 id="portfolio">Projects at Ada</h1>
+  <section id="projects_ada">
+    <div class="projects">
+      <h3>Viewing party</h3>
```

Comment:
Try to avoid skipping heading numbers - `h1` should be for the broadest, then `h2` then `h3` etc. If You need to adjust the sizes of them, do that in CSS instead of using an out-of-order heading. This makes it work much better with screen readers!